src/App.js

```diff
@@ -1,46 +1,67 @@
 import React, { Component } from 'react';
 import './App.css';
 import MoviesContainer from './MoviesContainer';
-import movieData from './testData.js';
 import InfoPage from './InfoPage';
+import { getMovies } from './apiCalls';
+import Error from './Error';
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData.movies,
+      movies: [],
       isHomepage: true,
-      selectedMovie: {}
+      selectedMovieId: null,
+      isError: false,
+      errorMessage: '',
```

Comment:
Nice work keeping state pretty light and only keeping track of the selected movie's ID.  I think instead of having two things to keep track of errors, you could probably just stick with the errorMessage instate.  If there is an errorMessage, then render it on line 58.